apps/condo/domains/user/schema/GetAccessTokenByUserIdService.js

```diff
@@ -58,18 +61,17 @@ const GetAccessTokenByUserIdService = new GQLCustomSchema('GetAccessTokenByUserI
             },
             schema: 'getAccessTokenByUserId (data: GetAccessTokenByUserIdInput!): GetAccessTokenByUserIdOutput',
             resolver: async (parent, args, context, info, extra = {}) => {
-                const { data } = args
-                if (!uuidValidate(data.userId)) throw new GQLError(ERRORS.INVALID_USER_ID, context)
-
-                let accessToken, ttl
-
-                try {
-                    ({ accessToken, ttl } = await getAccessTokenForUser(data.userId))
-                } catch (e) {
-                    if (e.message.includes('refreshToken')) {
-                        throw new GQLError(ERRORS.REFRESH_TOKEN_EXPIRED, context)
-                    }
-                    throw new GQLError(ERRORS.ERROR_GETTING_ACCESS_TOKEN, context)
+                const { data: { userId, organizationId } } = args
+                if (!uuidValidate(userId)) {
+                    throw new GQLError(ERRORS.INVALID_USER_ID, context)
+                }
+                if (!uuidValidate(organizationId)) {
+                    throw new GQLError(ERRORS.INVALID_ORGANIZATION_ID, context)
+                }
```

Comment:
Are these checks needed? Is checking by schema (ID!) not enough?

Reply:
Done